get-started/index.md

```diff
@@ -157,11 +172,6 @@ To completely remove Cryostat and all objects and recordings created by it:
     - If the Cryostat Operator has already been uninstalled, please reinstall it
       before deleting the Cryostat custom resource.
 3. Uninstall the Cryostat Operator.
```

Comment:
I would just add a caution here that this procedure removes the `my-cryostat-operator` namespace and its contents.

If the user deploys their applications there so Cryostat can work with them, they might not expect this command to also delete their applications.